streamz/tests/test_kafka.py

```diff
@@ -87,7 +87,8 @@ def kafka_service():
                 "Kafka not available. "
                 "To launch kafka use `export STREAMZ_LAUNCH_KAFKA=true`")
 
-        producer = ck.Producer({'bootstrap.servers': 'localhost:9092'})
+        producer = ck.Producer({'bootstrap.servers': 'localhost:9092',
+                                'topic.metadata.refresh.interval.ms': '5000'})
```

Comment:
Is this required? It's not in the docstrings that you need to set it.

Reply:
This is required in the producer configs for the producer to start producing to the newly added partitions sooner than the default topic metadata refresh interval (which is 5 minutes). This config is mainly so that the test doesn't need to wait for more than a few seconds for the partitions metadata to refresh.